sdk/storage/azure_storage_blob/src/clients/append_blob_client.rs

```diff
@@ -32,12 +32,12 @@ impl GeneratedAppendBlobClient {
     ///
     /// # Arguments
     ///
-    /// * `blob_url` - The full URL of the Append blob, for example `https://myaccount.blob.core.windows.net/mycontainer/myblob`.
+    /// * `append_blob_url` - The full URL of the Append blob, for example `https://myaccount.blob.core.windows.net/mycontainer/myblob`.
     /// * `credential` - An optional implementation of [`TokenCredential`] that can provide an Entra ID token to use when authenticating.
     /// * `options` - Optional configuration for the client.
     #[tracing::new("Storage.Blob.AppendBlob")]
     pub fn from_url(
-        blob_url: Url,
+        append_blob_url: Url,
```

Comment:
I think you may have misinterpreted a comment of mine. I think this should be just `blob_url` in all clients. You just had the doc string wrong in Block Blob client.